app/controllers/repositories_controller.rb

```diff
@@ -109,8 +109,12 @@ def destroy_info
   end
 
   def destroy
-    @project.repository.destroy
-    flash[:notice] = I18n.t('repositories.delete_sucessful')
+    repository = @project.repository
+    if repository.destroy
+      flash[:notice] = I18n.t('repositories.delete_sucessful')
+    else
+      flash[:error] = repository.errors.full_messages.join("\n")
```

Comment:
I could not believe that `flash` was clever enough to turn `\n` into actal HTML line breaks, so I tried it out:

````ruby
flash[:notice] = ['a', 'b', 'c'].join("\n")
````

Sadly the result was as I expected:
![image](https://cloud.githubusercontent.com/assets/453584/9629688/da67c4d8-5174-11e5-8ea7-b3446849b4ab.png)

**tl; dr:** You should either use `<br>` or some other clever trick... I believe in error flashs we are using unordered lists...